renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -2271,15 +2348,16 @@ void WrappedOpenGL::AttemptCapture()
     m_SuccessfulCapture = true;
     m_FailureReason = CaptureSucceeded;
 
-    m_ContextRecord->LockChunks();
-    while(m_ContextRecord->HasChunks())
+    GLResourceRecord *contextRecord = GetContextRecord();
```

Comment:
This will need to iterate over all context records and clear them all, since we'll only hit `AttemptCapture()` on the one thread but we want all the records to be cleaned each time.